src/main/resources/view/MainWindow.fxml

```diff
@@ -53,6 +53,14 @@
           <StackPane fx:id="staffListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
         </VBox>
 
+        <VBox fx:id="applicantList" styleClass="pane-with-border" minWidth="340" prefWidth="340" VBox.vgrow="ALWAYS">
+          <padding>
+            <Insets top="10" right="10" bottom="10" left="10" />
+          </padding>
+          <StackPane fx:id="applicantListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
+        </VBox>
+
+
```

Comment:
I guess we need to implement window/tab switching

Reply:
Yes, I just added it like this as of now, will adapt once new ui is pushed.